src/pages/Transaction/IssueTransaction/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { useParams } from 'react-router-dom';
+import { useErrorHandler, withErrorBoundary } from 'react-error-boundary';
+import { useQuery } from 'react-query';
+
+import ErrorFallback from 'components/ErrorFallback';
+import PrimaryColorEllipsisLoader from 'components/PrimaryColorEllipsisLoader';
+import { URL_PARAMETERS } from 'utils/constants/links';
+import issuesFetcher, { ISSUES_FETCHER } from 'services/fetchers/issues-fetcher';
+
+// http://localhost:3000/transaction/issue/0xfd6d53d8df584d675fe2322ccb126754d6c6d249878f0a2c9526607458714f76
+const IssueTransaction = (): JSX.Element => {
+  const { [URL_PARAMETERS.TRANSACTION_HASH]: transactionHash } = useParams<Record<string, string>>();
+
+  const {
+    isIdle: issueIdle,
+    isLoading: issueLoading,
+    data: issue,
+    error: issueError
+    // TODO: should type properly (`Relay`)
+  } = useQuery<any, Error>(
+    [
+      ISSUES_FETCHER,
+      0, // offset
+      1, // limit
+      `id_eq: "${transactionHash}"` // `WHERE` condition
+    ],
+    issuesFetcher
+  );
+  useErrorHandler(issueError);
+
+  if (issueIdle || issueLoading) {
+    return <PrimaryColorEllipsisLoader />;
+  }
+
+  console.log('[IssueTransaction] issue => ', issue);
```

Comment:
For now, it's just a prototype of a single issue transaction.

Reply:
I would suggest to also handle case when issue transaction with given `transactionHash` does not exist, but I think that can be also added in follow-up PR if you prefer to do it that way :)

Reply:
@peterslany 
I'm sorry. I don't understand. Could you please rephrase it?

Reply:
@crypto-engineer ah sorry. I think it would be useful to handle case when `useQuery` call returns empty result (when transaction with transactionHash from URL does not exist). I meant something like `404: Not found` when the URL param `transactionHash` is invalid.

 But if this component is just a prototype that will be changed then it doesn't need to be handled in this PR :)

Reply:
Thank you for rephrasing. I understand. :)
Yes, it's just a prototype for now so exception handling could follow after that.